test/platform/test_linux_amd64_compatibility.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+BLUE_FG=$(tput setaf 4)
+GREEN_FG=$(tput setaf 2)
+RED_FG=$(tput setaf 1)
+TEAL_FG=$(tput setaf 6)
+END_FG_COLOR=$(tput sgr0)
+
+OS_LIST=(
+    centos:7
+    centos:8
+    fedora:28
+    ubuntu:16.04
+    ubuntu:18.04
+)
+
+FAILED=()
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+PLATFORM=$($SCRIPTPATH/../../scripts/osarchtype.sh)
+
+if [ "${PLATFORM}" != "linux/amd64" ]
+then
+    echo "The test_linux_amd64_compatibility.sh script is intended to support local execution only on linux/x86-64 machines."
```

Comment:
Do you mind using the color vars such as `RED_FG` here?  I find it makes it much easier to spot in any log files.